src/main/java/nl/tudelft/model/bubble/Bubble.java

```diff
@@ -54,21 +54,29 @@
      *            boolean - true if the bubble should initialise moving to the right.
      * @param resources
      *            {@link ResourcesWrapper} - The resources that this class may use.
+     * @param bubbleFactoryFactory
+     *            {@link BubbleFactoryFactory} - The BubbleFactoryFactory that this class may use.
      */
     public Bubble(Image bubbleImg, float locX, float locY, boolean goRight,
-            ResourcesWrapper resources) {
```

Comment:
I think the boolean goRight can be removed, seeing as it always defaults to true in the bubblefactory.